docs/src/content/docs/de/guides/sidebar.mdx

```diff
@@ -298,22 +349,23 @@ Die obige Konfiguration erzeugt die folgende Seitenleiste:
 
 ### Abzeichenvarianten
 
-Passe das Design des Abzeichens an, indem du ein Objekt mit den Eigenschaften `text` und `variant` verwendest.
+Passe das Design des Abzeichens mit einem Objekt mit den Eigenschaften `text`, `variant` und `class` an.
 
 Der `text` steht für den anzuzeigenden Inhalt (z.B. "Neu").
 Überschreibe das `default`-Styling, das die Akzentfarbe deiner Website verwendet, indem du die Eigenschaft `variant` auf einen der folgenden Werte setzt: `note`, `tip`, `danger`, `caution` oder `success`.
 
-```js
+Optional kannst du einen benutzerdefinierten Abzeichenstil erstellen, indem du die Eigenschaft `class` auf einen CSS-Klassennamen setzt.
+
+```js {9}
 starlight({
 	sidebar: [
 		{
-			label: 'Anleitungen',
+			label: 'Sterne',
 			items: [
-				// Ein Link mit einem gelben "Experimentell"-Abzeichen.
+				// Ein Link mit einem gelben „Stub“-Abzeichen.
 				{
-					label: 'Komponenten',
-					link: '/guides/components/',
-					badge: { text: 'Experimentell', variant: 'caution' },
+					slug: 'stars/sirius',
+					badge: { text: 'Stub', variant: 'caution' },
```

Comment:
Just to double check, could "Stub" have been missed, just the same translation or do you think it's better to keep as-is?

Reply:
Im not sure what the word "Stub" means in English, so I actually dont know if there is a good translation in German. Im going to ask some other German translators on this one...

Reply:
Sure, don't know if it helps but Wikipedia seems to use ["Lückenhaft"](https://de.wikipedia.org/wiki/Wikipedia:L%C3%BCckenhaft).

Reply:
Ah, so "Stub" means that the page is not yet fully written and therefore there can be parts missing in the documentation? This checks out, I'll change that thx!

Reply:
Okay, thanks to @delucis I have no "chosen" (it got suggested on [Discord](https://discord.com/channels/830184174198718474/972142807869423617/1295516459417731123)) the translation "Platzhalter" for "Stub" which fits very nicely in this sence for the German language.